### PR TITLE
Align with osp-d (neutron-related)

### PR DESCRIPTION
**Pull request by cwolferh**

### CODE REVIEW

File: puppet/modules/quickstack/manifests/pacemaker/neutron.pp [hunk 1]
```diff
@@ -250,22 +250,16 @@
...
-      notify { 'PLUMgrid neutron-scale resource':
+      notify { 'PLUMgrid core plugin':
         require => Quickstack::Pacemaker::Resource::Generic['neutron-server'],
       }
       ->
       Anchor['pacemaker ordering constraints begin']
 
     } else {
-      notify { 'neutron-scale resource':
+      notify { 'not PLUMgrid core plugin':
```
Comment (jguiditta) [inline puppet/modules/quickstack/manifests/pacemaker/neutron.pp]: Why not just drop this notify?

**Comment by jguiditta** [comment]
I may be missing something here, but don't we want a neutron-server -> neutron-openvswitch-agent order constraint in here?

File: puppet/modules/quickstack/manifests/pacemaker/neutron.pp (hunk 1 above)
Comment (cwolferh) [inline puppet/modules/quickstack/manifests/pacemaker/neutron.pp]: I'm not saying that is a bad thing necessarily, but that feels like more of a refactor and out of scope of this PR.  The notify's are both a little awkward to begin with IMO, but still I'd rather not touch them here.

**Comment by cwolferh** [comment]
Good catch, I've added that.

**Comment by cwolferh** [comment]
"pcs constraint show" now includes:

    start neutron-server-clone then start neutron-openvswitch-agent-clone (kind:Mandatory)

and

    start neutron-ovs-cleanup-clone then start neutron-netns-cleanup-clone (kind:Mandatory)
    start neutron-netns-cleanup-clone then start neutron-openvswitch-agent-clone (kind:Mandatory)
    start neutron-openvswitch-agent-clone then start neutron-dhcp-agent-clone (kind:Mandatory)

**Comment by jguiditta** [comment]
I see the same on my setup, and NeutronScale is not missing as expected, good to merge!

**Pull request merged**